test/unit/tag_test.rb

```diff
@@ -17,6 +17,13 @@ class TagTest < ActiveSupport::TestCase
     assert followers.include?(tag_selections(:awesome).user)
   end
 
+  test 'related tags' do
+    related = Tag.related(tags(:awesome).name)
+    assert !related.empty?
+    assert related.include?(tags(:test))
+    puts related.to_json
```

Comment:
```suggestion
```
I don't think that anything is to be printed here. So, there is no need for this statement.